### ISSUE TITLE
[CoreBundle] WinzouStateMachine -> sylius_order.yml -> wrong wording

### ISSUE THREAD

**Issue by AymLaf** [Sylius/Sylius#13688]
https://github.com/Sylius/Sylius/blob/1b0aa43ae8d2983a0c29675fbdb4398dafebdb56/src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order.yml#L95

Instead of `sylis_cancel_order` it shoud be `sylius_cancel_order`

**Comment by vvasiloi** [comment]
Thank you @AymLaf! There's an open PR to fix this issue: https://github.com/Sylius/Sylius/pull/12762!

**Comment by Rafikooo** [comment]
Hi Folks! Yes, this typo hurts the eye, however, the problematic of the change due to the BC policy is too big compared to the benefit of no typo, so we decided to abandon this topic until the release of Sylius 2.0 💃

**Comment by SVillette** [comment]
Hi, in my opinion, 2.0 is too far for an issue like this. In our project, it caused a bug where product onHand were updated twice because when we wanted to override this event, we fixed it thinking that the typo came from us.
So instead of overriding this event, we just duplicated it. I think this error can happen in other projects.

Once again a fix in 2.0 does not seem legit to me as it could be easily added in UPGRADE-1.x for a minor version and 2.0 does not have any plans about an upcoming release.
